### PR TITLE
Created the appointment controller for Account

### PR DESCRIPTION
**Pull request by Theolem**
show, create/new, destroy and index.

### CODE REVIEW

File: app/controllers/account/appointments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+module Account
+  class AppointmentController < ApplicationController
```
Comment (AdrienLiagre) [inline app/controllers/account/appointments_controller.rb]: You forgot the `s` for the name of the controller

File: app/controllers/account/appointments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    def create
+      @appointment = current_user.appointments.new(appoint_params)
+      if @appointment.save
+        redirect_to root_path
+      else
+        render :new
+      end
+    end
+
+    def destroy
```
Comment (AdrienLiagre) [inline app/controllers/account/appointments_controller.rb]: I think you just copied colled some part of the girlfriends controller without modify it ..... Be more focused next time pls. Time is money

**Comment by AdrienLiagre** [comment]
Review done

**Comment by Theolem** [comment]
Fixing done

File: app/controllers/account/appointments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      @appointment = @current_user.appointments.find(params[:id])
```
Comment (AdrienLiagre) [inline app/controllers/account/appointments_controller.rb]: you should'nt put @current_user, but current_user

**Comment by AdrienLiagre** [comment]
2nd review done

**Pull request merged**